org.jacoco.doc/pom.xml

```diff
@@ -127,7 +188,7 @@
             <configuration>
               <reportOutputDirectory>${project.build.directory}/apidocs</reportOutputDirectory>
               <includeDependencySources>true</includeDependencySources>
-              <excludePackageNames>*.internal,org.jacoco.ant</excludePackageNames>
+              <excludePackageNames>*.internal,org.jacoco.ant,org.jacoco.maven</excludePackageNames>
```

Comment:
We also should exclude `org.jacoco.examples` to not make it part of our API documentation.